src/main/java/no/nav/familie/integrasjoner/journalpost/HentJournalpostController.kt

```diff
@@ -35,6 +33,15 @@ class HentJournalpostController(private val journalpostService: JournalpostServi
                 .body(failure(errorBaseMessage + errorExtMessage, error = ex))
     }
 
+    @ExceptionHandler(JournalpostForbiddenException::class)
+    fun handleJournalpostForbiddenException(e: JournalpostForbiddenException): ResponseEntity<Ressurs<Any>> {
+        LOG.warn("Bruker eller system ikke tilgang til saf ressurs: ${e.message}")
+
+        return ResponseEntity
+                .status(HttpStatus.OK)
```

Comment:
Her burde vi vel returnere noe annet enn OK?

Reply:
Hvorfor det? Da må vi håndtere 403 error i ba/ef-sak. Ref https://nav-it.slack.com/archives/CJN0STWB0/p1624287128110400